articles/machines/backend-simulators.md

```diff
@@ -26,6 +26,26 @@ IonQ provides a GPU-accelerated idealized simulator supporting up to 29 qubits,
 
 For more information, see the [IonQ provider](xref:microsoft.quantum.providers.ionq) page.
 
+## QCI
+
+The QCI simulator is designed to model QCI hardware. Although the simulator is
+ideal with regard to quantum noise, it accurately reflects precision and
+accuracy of classical calculations on 18 bit signed arithmetic on integers and
+fixed-point registers. Before simulating, it completes a full compilation and
+thereby validates the program against hardware constraints.
+
+The QCI simulator currently supports 8 qubits in a ring topology and will let you
+simulate up to X shots. The simulator is always available and never in a degraded
+or unavailable state. Contact support@quantumcircuits.com for help.
+
+<!-- ADD ANY CONTENT THAT YOU WANT TO USE TO DESCRIBE YOUR SERVICES AND COMPANY -->
+
+- Job type: `Simulation`
+- Data Format: `microsoft.qir.v1`
+- Target ID: `qci.simulator`
```

Comment:
@Bradben should we add "Publisher" or "Provider ID"?
cc @qci-petrenko

Reply:
Since you won't have a dedicated provider page yet, that's a good idea. 

Publisher: [QCI](https://www.quantumcomputinginc.com/)
Provider ID: `qci`

Assuming that is the page you want to link to.

Reply:
This may be resolved, but GH isn't letting me do so.